includes/utilities.inc

```diff
@@ -17,21 +17,38 @@ function islandora_usage_stats_log_object_view(AbstractObject $object) {
     if (($last_viewed + variable_get('islandora_usage_stats_cooldown_seconds', 300)) < REQUEST_TIME) {
       module_load_include('inc', 'islandora_basic_collection', 'includes/utilities');
       module_load_include('inc', 'islandora_usage_stats', 'includes/db');
-      $pid_id = islandora_usage_stats_get_pid_id($object->id);
-      $object_access_id = islandora_usage_stats_object_view_to_db($pid_id);
+      try {
+        $pid_id = islandora_usage_stats_get_pid_id($object->id);
+        $object_access_id = islandora_usage_stats_object_view_to_db($pid_id);
+      }
+      catch (Exception $e) {
+        watchdog('islandora_usage_stats', 'Error: islandora_usage_stats_pid_to_db() could not load %pid', array('%pid' => $object->id), WATCHDOG_ERROR);
```

Comment:
Ah yes, this is the only one that I can't even test since the object getting passed to `islandora_usage_stats_pid_to_db()` is the original object getting passed into `islandora_usage_stats_log_object_view()` so in practice, `islandora_usage_stats_log_object_view()` should croak on invocation (expected object, got FALSE) before it ever gets to this try block.

Even though that first try block should never even get to execution if the object passed to `islandora_usage_stats_log_object_view()` is FALSE, I just put it there for consistency with the other two calls to `islandora_usage_stats_pid_to_db()`. Since this is the case, should I just ditch the try/catch blocks for this one and leave the other two in place, or do you have another suggestion?